Pod/Classes/UI/CardTextField.swift

```diff
@@ -283,6 +309,18 @@ public class CardTextField: UITextField, NumberInputTextFieldDelegate {
         monthTextField?.deleteBackwardCallback = {_ -> Void in self.numberInputTextField?.becomeFirstResponder()}
         yearTextField?.deleteBackwardCallback = {_ -> Void in self.monthTextField?.becomeFirstResponder()}
         
+        if isRightToLeftLanguage {
+            cvcTextField.textAlignment = .Left
+            monthTextField.textAlignment = .Left
+            yearTextField.textAlignment = .Right
+            numberInputTextField.textAlignment = .Right
+        } else {
+            cvcTextField.textAlignment = .Right
+            monthTextField.textAlignment = .Right
+            yearTextField.textAlignment = .Left
+            numberInputTextField.textAlignment = .Left
```

Comment:
cf Apple documentation: 

> Use the default alignment associated with the current localization of the app. The default alignment for left-to-right scripts is NSTextAlignmentLeft, and the default alignment for right-to-left scripts is NSTextAlignmentRight.

So it should work if your app is using a language that requires a RTL alignment right?